Client/Frontend/Strings.swift

```diff
@@ -2558,7 +2558,7 @@ extension String {
     public static let ContextMenuButtonToastNewTabOpenedLabelText = MZLocalizedString(
         "ContextMenu.ButtonToast.NewTabOpened.LabelText",
         tableName: nil,
-        value: "New Tab opened",
+        value: "New Tab Opened",
```

Comment:
We need to adapt the key as well for this change to be effectively exported for string translations. So `"ContextMenu.ButtonToast.NewTabOpened.LabelText.v114"` would work